boards/lilygo/ttgo_t8s3/doc/index.rst

```diff
@@ -0,0 +1,247 @@
+.. _ttgo_t8s3:
```

Comment:
Can you update this document to follow latest changes? Ex.:
1) `.. zephyr:board:: ttgo_t8s3`